### DISCUSSION TITLE
X-Plane 12 crashes with this screen

### DISCUSSION THREAD

**Discussion by geronimopilot** [kubilus1/autoortho#152] [Q&A]
Hello,

Can anyone explain what this screen shot is telling me? I believe I have downloaded everything that is needed but no matter what I try this same screen appears, then XP12 shuts down.

Thank for you help :)
Scott
![autoortho_error](https://github.com/kubilus1/autoortho/assets/134178184/afec4cf8-b990-46ff-8cad-3e6ce25fcac7)

**Comment by TinkerNZ** [comment]
Hi Scott,
Are you using Symlinks by chance?  I moved my base files (z_na_##) and symlinked them back to Custom Scenery in XP and got the same sort of error and crash of XP12.
I changed the "X_Plane scenery dir." to the location where the actual files are located (instead of X-Plane12/custom scenery) and now things work OK for me. I do get a Warning on startup from AO that things may not work, but that can just be clicked (OK) away and it runs.

TinkerNZ

**Comment by geronimopilot** [comment]
TinkerNZ,

No sorry but I was able to successfully launch AO! I needed to "run" AO prior to launching XP.....

Scott
